### DISCUSSION TITLE
Can a meeting says it's a group but not belong to a district?

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#709] [Q&A] [answered]
TMSL v3.13 added a fix to the dropdown list.  See #586 which brought up the issue of when someone added or edited a meeting and selected group it defaulted to the word "District" if one wasn't selected from the dropdown list. 

But now if I don't select a district from the dropdown, it defaults to "None" and displays it as blank. 

Shouldn't the admin have to select one of the districts from the dropdown? Can a meeting says it's a group but not belong to a district? I'm not so sure that's the way it works, but TSML v3.13 allows that to happen.

What it doesn't allow is to select **Part of a group**, but not enter a group name. In that case radio button switches from **Part of a group** to **Individual meeting** when I try to save it. And that is the way it should be. But why isn't that the case also for defaulting to None for District from the dropdown list?

I would like to suggest an error flag come up when someone tries to save a meeting entry that doesn't select from the District dropdown to alert them they must, or revert back to Individual Meeting. And perhaps also add flag when someone tries to save a meeting entry that doesn't have a group name entered even though they selected **Part of a group**. Right now, it just reverts back to Individual Meeting without any explanation.

**Comment by kiyote33** [comment]
Absolutely. Per AA tradition 4 ("Each group should be autonomous...") groups may choose to be part of AA's service structure or not.

**Reply by Grateful-Viv** [reply]
My understanding is one of the main differences between a meeting and a group is that groups do elect to participate in A.A.'s structure. 

By registering with GSO, the group then has a vote in the decision making process of A.A. as a whole. Otherwise, it's just a meeting. 

I realize this discussion goes beyond the scope of TSML plugin, so I will drop it.

**Comment by kiyote33** [comment, marked as answer]
I've heard others suggest the same thing in the past. When we approached GSO about it at our region forum, the reply we got is that they have no strict definition differentiating a group and meeting.  A Group usually has a regular scheduled meeting, but GSO has no interest in dictating this definition. Groups, districts, and areas are free to decide for themselves on this matter.

Code4Recovery's plugin is being used by other recovery programs, so we need to be mindful of putting logic in place which has rules which might not be supported by all, now and in the future.

**Reply by Grateful-Viv** [reply]
I'm good with that. I know GSO also has made it much easier for natively online groups to register and decide for themselves what (and I guess now, if any) districts/area they want to be part of. You can see all this on their new group form [here](https://www.aa.org/sites/default/files/literature/F30_NewGroupListingGuidelinesForm_fillable_EN.pdf).

**Reply by kiyote33** [reply]
Thx for the link to the new form. It does seem to support what I just said, but for those groups wishing to be part of the AA service structure here in North America, it implies support for your definition. Typical of GSO, they don't want to be the source for a hard & fast answer to the question.  That form does at least try to provide some guidance for AA Groups/meetings in general.

**Reply by Grateful-Viv** [reply]
They also have a [new form](https://www.aa.org/sites/default/files/literature/F28_GroupInformationChangeForm_fillable_EN.pdf) to fill out for any group changes.
